infrastructure/src/main/java/mops/database/dto/AuswahlDto.java

```diff
@@ -0,0 +1,19 @@
+package mops.database.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@AllArgsConstructor
+@Data
+@Table("auswahl")
+public class AuswahlDto {
+  @Id
+  Long id;
+  String auswahltext;
+
+  public static AuswahlDto create(String text) {
+    return new AuswahlDto(null, text);
+  }
```

Comment:
This looks a little bit like a constructor. The values are filled automatically by Spring Data JDBC

Reply:
I followed this tutorial https://www.youtube.com/watch?v=EaHlancPA14  
It says that you should use a create method to insert values into the database. This way the Id will also be incremented automatically.

Reply:
Ah, ok.